controllers/release/adapter.go

```diff
@@ -67,10 +67,10 @@ func newAdapter(ctx context.Context, client client.Client, release *v1alpha1.Rel
 	}
 
 	releaseAdapter.validations = []controller.ValidationFunction{
+		releaseAdapter.validatePipelineDefined,
 		releaseAdapter.validateProcessingResources,
 		releaseAdapter.validateAuthor,
 		releaseAdapter.validatePipelineRef,
```

Comment:
It would be great to rename this so it doesn't sound like it's similar to the new one.

Reply:
This *is* the new one. Which one do you not want to sound similar to? Then I can adjust

Reply:
`validatePipelineRef` and `validatePipelineDefined`.

Reply:
Changed `validatePipelineRef` to `validatePipelineSource` as it is checking that the pipeline is from a bundle or git repo vs cluster scoped